### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by machinesmith42**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,33 @@
...
+# Using FREX
+
+Add the maven repo where my libraries live to your build.gradle
+
+```gradle
+repositories {
+    maven {
+    	name = "dblsaiko"
+    	url = "https://maven.dblsaiko.net/"
+    }
```
Comment (spiralhalo) [inline README.md]: We use vram maven now

**Comment by machinesmith42** [comment]
@spiralhalo  Is that the correct maven?

**Comment by spiralhalo** [comment]
> 
> 
> @spiralhalo Is that the correct maven?

It should be. @grondag

**Comment by grondag** [comment]
The maven is right but the dependency groups are not.  Still better than it was.  Will fix on my end.

**Pull request merged**